packages/cli/test/functional/test_site_templates/test_portfolio/expected/index.md

```diff
@@ -0,0 +1,70 @@
+<frontmatter>
+  title: Portfolio
+  layout: default.md
+</frontmatter>
+
+<br>
+
+<div class="bg-light text-black px-2 py-5 mb-4">
+  <div class="container">
+    <h1 class="display-5 no-index"><md>:wave:</md> Hello!<br>I'm {{ name }}</h1>
+    <p class="lead">Welcome to my portfolio!</p>
+    <div style="margin-bottom:20px;">
+      <span style="margin-right:10px;">
+        <a href="https://github.com/MarkBind/markbind" target="_blank" class="icon">
+          <i class="fa-brands fa-github fa-2xl"></i>
+        </a>
+      </span>
+      <span style="margin-right:10px;">
+        <a href="https://www.linkedin.com/school/national-university-of-singapore/" target="_blank" class="icon">
+          <i class="fa-brands fa-linkedin fa-2xl"></i>
+        </a>
+      </span>
+      <span style="margin-right:10px;">
+        <a href="mailto:secretary_cs@comp.nus.edu.sg" target="_blank" class="icon">
+          <i class="fa-solid fa-envelope fa-2xl"></i>
+        </a>
+      </span>
+    </div>
+    <a class="btn btn-primary" href="./contents/assets/ug_template.docx" download="UG-template-resume" role="button">Resume</a>
```

Comment:
Perhaps using Download Resume could be more indicative to the viewer that it is a button to download as my expected intention when clicking the button is to be brought to a page showing the resume. Another way to go about this could be to exporting the docx file to pdf to allow it to be opened in a new tab instead of forcing a download. Imo using the pdf would be my preference

Reply:
Yep makes sense! I've changed it to the pdf file, you can check it again here: https://portfolio-template-markbind.netlify.app/